### ISSUE TITLE
No projects files on call

### ISSUE THREAD

**Issue by NormandoHall** [nextcloud/spreed#1945]
## Steps to reproduce
1. Attach file to share with a talk
2. Click Projects and the files are there
2. Call
3. Click Projects and file are gone

### Expected behaviour
Files must be there

### Actual behaviour
Files no there

## Browser

**Microphone available:** yes

**Camera available:** yes/no

**Operating system:** Windows

**Browser name:** Firefox/Chrome/ both

**Browser version:** 50.1/55/... meet requierements

### Browser log

<details>

```
merged.js?v=2d255e61-34:8391 TypeError: Cannot read property 'typeString' of undefined
    at Object.getLabel (collaboration.js:67)
    at i (merged.js?v=2d255e61-34:8479)
    at a.options (merged.js?v=2d255e61-34:8479)
    at hn.get (merged.js?v=2d255e61-34:8391)
    at hn.evaluate (merged.js?v=2d255e61-34:8391)
    at a.options (merged.js?v=2d255e61-34:8391)
    at a.render (merged.js?v=2d255e61-34:8479)
    at a.t._render (merged.js?v=2d255e61-34:8391)
    at a.o (merged.js?v=2d255e61-34:8391)
    at hn.get (merged.js?v=2d255e61-34:8391)
```

</details>

## Spreed app

**Spreed app version:** 6.0.2

## Server configuration

**Operating system**: RedHat

**Web server:** Nginx

**Database:** MySQL

**PHP version:** 7.3

**Nextcloud Version:** 16.0.1

**List of activated apps:**

<details>

```
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - announcementcenter: 3.5.1
  - audioplayer: 2.7.2
  - bookmarks: 1.0.6
  - bruteforcesettings: 1.3.0
  - calendar: 1.7.0
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - contacts: 3.1.3
  - dav: 1.9.2
  - event_update_notification: 0.3.4
  - external: 3.3.0
  - extract: 1.2.0
  - federatedfilesharing: 1.6.0
  - files: 1.11.0
  - files_antivirus: 2.1.1
  - files_ebookreader: 0.0.1
  - files_fulltextsearch: 1.3.2
  - files_fulltextsearch_tesseract: 1.3.0
  - files_pdfviewer: 1.5.0
  - files_rightclick: 0.13.0
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - firstrunwizard: 2.5.0
  - fulltextsearch: 1.3.4
  - fulltextsearch_elasticsearch: 1.3.4
  - gallery: 18.3.0
  - groupfolders: 4.0.3
  - impersonate: 1.3.0
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - nextcloud_announcements: 1.5.0
  - notes: 3.0.0
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - quicknotes: 0.1.7
  - quota_warning: 1.5.0
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - spreed: 6.0.2
  - support: 1.0.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - tasks: 0.11.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - updatenotification: 1.6.0
  - viewer: 1.0.0
  - workflowengine: 1.6.0
Disabled:
  - admin_audit
  - encryption
  - federation
  - files_external
  - files_retention
  - previewgenerator
  - recommendations
  - user_ldap
```

</details>

**Nextcloud configuration:**

<details>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "drive.xxx.com.ar",
            "xxx.xxx.xxx.xxx"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "16.0.1.1",
        "overwrite.cli.url": "https:\/\/drive.xxx.com.ar",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mysql.utf8mb4": true,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\HEIC",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\Illustrator",
            "OC\\Preview\\Movie",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\PDF",
            "OC\\Preview\\Photoshop",
            "OC\\Preview\\Postscript",
            "OC\\Preview\\StarOffice",
            "OC\\Preview\\SVG",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\Font"
        ],
        "default_language": "es",
        "filelocking.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "maintenance": false,
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable"
    }
}

```
</details>

### Server log (data/nextcloud.log)

<details>

```
Insert your server log here
```

</details>

**Comment by nickvergessen** [comment]
Mind to add some screenshots to help us reproduce your issue?
The steps to reproduce are not working like that for me.

**Comment by NormandoHall** [comment]
> Mind to add some screenshots to help us reproduce your issue?
> The steps to reproduce are not working like that for me.

Yes, sure.

STEP 1 (previous call project fijes are there):
![image](https://user-images.githubusercontent.com/2943127/60607315-63b3ec80-9d93-11e9-9da9-2422f043fc2b.png)

STEP 2 (calling):
![image](https://user-images.githubusercontent.com/2943127/60607441-a1b11080-9d93-11e9-9290-a902911fe9b4.png)

STEP 3 (in call, when I select Projects tab files are gone and get this console error:
![image](https://user-images.githubusercontent.com/2943127/60607514-cb6a3780-9d93-11e9-829e-709666575f63.png)

**Comment by nickvergessen** [comment]
@juliushaertl can you have a look and help here?

1. Create a new room
2. Add a file to the room as a project
3. Start a call (doing it alone is already enough)
4. Switch to projects tab :boom:

**Comment by juliushaertl** [comment]
Fixed in the nextcloud-vue-collections library https://github.com/juliushaertl/nextcloud-vue-collections/releases/tag/v0.5.5

I'll trigger a dependabot update so we get the new version bump pull request.

**Comment by danxuliu** [comment]
Fixed in #2044
